database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -9,7 +9,7 @@
     public function up(): void
     {
         Schema::create('users', function (Blueprint $table) {
-            $table->id();
+            $table->id()->primaryKey();
```

Comment:
It's not necessary to call `->primaryKey()` here, because `->id()` aldready sets that column as a primary key in the db.